### ISSUE TITLE
API ML Startup Message Duplication

### ISSUE THREAD

**Issue by dkelosky** [zowe/zowe-install-packaging#4194]
After an attempt to startup zowe server-side components, I saw several messages that appeared to be duplicates in my job log:

```text
+ZWEL0021I Zowe Launcher starting
+ZWEL0018I Zowe instance prepared successfully
+ZWEL0006I starting components
+ZWEL0001I component zss started
+ZWEL0001I component gateway started
+ZWEL0001I component discovery started
+ZWEL0001I component api-catalog started
+ZWES1013I ZSS Server has started. Version '2.12.0+20231016' 31-bit
...
```

Messages `ZWEL0001I` use lower-case messages while the others use capitalization.  

Similarly, the components are not referred to consistently by the same name in these messages `zss` and `ZSS Server`.  

Lastly, there are two times that the `ZSS Server` component has a message that it "started".  

Should these startup messages be cleaned up for consistency?

**Comment by balhar-jakub** [comment]
Probably, it seems like you believe that they should be consistent. Do I get it correctly?

**Comment by dkelosky** [comment]
Yes consistent on naming, capitalization, and in conveying the accurate status.  Thank you
